src/app/inventory/store/stats.ts

```diff
@@ -185,7 +185,7 @@ function buildClassItemStatsFromMods(
   }
 
   for (const socket of modSockets) {
-    if (socket?.plugged?.stats) {
+    if (socket.plugged?.enabled && socket?.plugged?.stats) {
```

Comment:
You don't need all the ?. in the second condition because it's guaranteed that `socket.plugged` exists at that point.

Reply:
Done, missed that.